packages/manager/apps/order-tracking/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "@ovh-ux/order-tracking-app",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "build": "webpack",
+    "dev": "webpack-dev-server",
+    "dev:watch": "yarn run dev",
+    "start": "lerna exec --stream --scope='@ovh-ux/order-tracking-app' --include-filtered-dependencies -- npm run build --if-present",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/order-tracking-app' --include-filtered-dependencies -- yarn run dev",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/order-tracking-app' --include-filtered-dependencies -- yarn run dev:watch"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-config": "^0.4.0",
+    "@ovh-ux/manager-core": "^7.1.3",
+    "@ovh-ux/ng-ovh-http": "^4.0.2",
+    "@ovh-ux/ng-ovh-sso-auth": "^4.1.0",
+    "@ovh-ux/order-tracking": "^0.0.0",
```

Due to the `BREAKING CHANGE`  and the release script, I think this is more suitable:

```suggestion
    "@ovh-ux/order-tracking": "^0.0.0 || ^1.0.0",
```